### ISSUE TITLE
QOL improvements to genders in notifications

### ISSUE THREAD

**Issue by mhetralla** [alucas/Snorlax#48]
* **Change the position of the gender**: Instead of *"♀ - Doduo - CP 63- Lvl 3,0"*, change it to *"Doduo ♀  - CP 63- Lvl 3,0"*.
* **Hide gender for Nidorans**: No need for getting duplicated gender symbols on them. (*"Nidoran♀ ♀  - CP 63- Lvl 3,0"*)
* **Change the symbol for genderless mons**: The symbol used right now shows as an emoji (small orange dot), I propose **⛒** instead. (☿ is the official unisex icon, but ⛒ is more recognizable) _or_ **Don't show anything for genderless mons.**

**Comment by alucas** [comment]
Yep, it's already like that in my Dev branch, juste need to filter Nidorans ;-)

**Comment by mhetralla** [comment]
Have you synced the commits? The dev branch is even with same as the master one. 😀

**Comment by alucas** [comment]
The dev branch is not sync yet ^^ I will do that as soon as possible
